nucypher/blockchain/eth/deployers.py

```diff
@@ -543,19 +546,19 @@ def deploy(self,
         # Switch the contract for the wrapped one
         the_escrow_contract = wrapped_escrow_contract
 
-        # 3 - Transfer the reward supply tokens to StakingEscrow #
-        reward_function = self.token_contract.functions.transfer(the_escrow_contract.address,
-                                                                 self.economics.erc20_reward_supply)
+        # 3 - Approve transfer the reward supply tokens to StakingEscrow #
+        approve_reward_function = self.token_contract.functions.approve(the_escrow_contract.address,
+                                                                        self.economics.erc20_reward_supply)
```

Comment:
I like this, specially considering the problem described in #1532. Just approving and not sending makes it even more secure and organized.